sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -541,10 +541,10 @@ export interface IndexingSchedule {
 }
 
 // @public
-export type IndexIterator = PagedAsyncIterableIterator<SearchIndex, SearchIndex[], {}>;
+export type IndexIterator = PagedAsyncIterableIterator<SearchIndex, SearchIndex[], Record<string, unknown>>;
```

Comment:
this one is more questionable, because the idea is that we don't support any pagination options, right? This would allow someone to pass in any object, perhaps expecting it to work. Can we make this `void` instead or does TS complain?

Reply:
In general, moving from `{}` to `Record<string, unknown>` or any map type is incorrect. `{}` is quite different from `object`.

Reply:
I have changed it back to {}